cmd/nodecmd/create.go

```diff
@@ -181,6 +188,11 @@ func createNodes(_ *cobra.Command, args []string) error {
 				publicIPMap[node] = cloudConfig.PublicIPs[i]
 			}
 		}
+		if separateMonitoringInstance {
+			if err = awsAPI.AddSecurityGroupRule(ec2Svc, cloudConfig.PublicIPs[len(cloudConfig.PublicIPs)-1], cloudConfig.SecurityGroup); err != nil {
```

Comment:
`[len(cloudConfig.PublicIPs)-1` we need a better way to get IP of the monitoring instance. I suggest using separate TF code and separate output.

Reply:
as we would need difference cpu/mem/disk for this instance